libcontainer/rootfs_linux.go

```diff
@@ -174,6 +175,10 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		if err != nil {
 			return err
 		}
+		flags := m.Flags
+		if flags&syscall.MS_RDONLY != 0 {
+			logrus.Warn("Cgroups are not mounted as read-only, container might affect host's resources.")
```

Comment:
I thinks so, it happens in child process. But we'll still get this warning, right?